extensions/wikia/Recirculation/services/DiscussionsDataService.class.php

```diff
@@ -33,7 +33,11 @@ function() {
 			}
 		);
 
-		$data = $this->formatData($rawData);
+		if ( $type === 'posts' ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "posts" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)